org.openfodder.OpenFodder.yml

```diff
@@ -0,0 +1,65 @@
+app-id: org.openfodder.OpenFodder
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm11
+build-options:
+  append-path: /usr/lib/sdk/llvm11/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm11/lib
+command: openfodder
+rename-appdata-file: openfodder.appdata.xml
+rename-desktop-file: openfodder.desktop
+rename-icon: openfodder
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-data
+
+modules:
+  - name: openfodder
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/OpenFodder/openfodder.git
+        tag: 1.6.0
+        commit: eea76b9d984cdcf62f39d00153dd30aed661cce5
+      - type: script
+        dest-filename: openfodder.sh
+        commands:
+          - export XDG_DATA_DIRS=/app/extra/:/app/share
+          - exec /app/share/OpenFodder/OpenFodder
+      - type: file
+        path: openfodder.desktop
+      - type: file
+        path: openfodder.png
+      - type: file
+        path: openfodder.appdata.xml
+    build-commands:
+      - make -j${FLATPAK_BUILDER_N_JOBS}
+    post-install:
+      - mkdir -p ${FLATPAK_DEST}/share/OpenFodder/
+      - cp -r  Run/* ${FLATPAK_DEST}/share/OpenFodder/
+      - install -Dm755 openfodder.sh ${FLATPAK_DEST}/bin/openfodder
+      - install -Dm644 openfodder.appdata.xml -t ${FLATPAK_DEST}/share/metainfo/
```

Comment:
as mentionned above, you might as well have all the names right from the get go.